README.md

```diff
@@ -39,6 +39,13 @@ Assuming that you're starting from **this directory**:
 1. To build and run the web app, Docker needs to know about some environment variables. There are examples ones in `dev.env` - copy it to `.env` and fill in the correct values for you. Save the file and then run `source .env`.
 1. Install [Docker](https://docker.com)
    1. Also, For windows, click [here](https://wslstorestorage.blob.core.windows.net/wslblob/wsl_update_x64.msi) to install the WSL 2 (Windows Subsystem for Linux, version 2)
+1. `cd docker`
```

Comment:
Wait a second, sorry, I think some duplicate lines crept in here from that merge conflict, and I didn't notice them earlier.

Reply:
@yemoski can you please remove the duplicate text? Then this will be good to go.